cmd/initialize/initlizeMonorepo.go

```diff
@@ -0,0 +1,290 @@
+		OrganizationId:     mc.OrganizationId,
+		AppName:            &newApp.Name,
+		AppAlternateId:     &newApp.AlternateId,
+		AppId:              &newApp.ID,
+	}
+
+	// Initialize only the config file in the app directory
+	err = manifest.InitializeConfig(mcl, filepath.Join(appDir, manifest.ManifestConfigFile))
+	if err != nil {
+		return err
+	}
+
+	// Create a manifest with the monorepo secret
+	ml := manifest.Manifest{
+		Config: mcl,
+		Secret: monorepoSecret,
+	}
+
+	// List environments for the project
+	environments, err := envService.ListEnvironments(orgID, *ml.ProjectId, 100, 1)
+	if err != nil {
+		return err
+	}
+
+	for _, e := range environments {
+		envName := strings.ToLower(e.Name)
+		envID := e.ID
+		err = CreateAndPushEmptyEnvFileMonorepo(cmd, envService, ml, orgID, newApp.ID, envID, envName, appDir)
+		if err != nil {
+			return err
+		}
+	}
+
+	err = CreateAndPushEmptyEnvFileMonorepo(cmd, envService, ml, orgID, newApp.ID, "default", "default", appDir)
+	if err != nil {
+		return err
+	}
+
+	err = CreateGitignoredFileMonorepo(cmd, filepath.Join(appDir, ".env.local"), ".env.local")
+	if err != nil {
+		return err
+	}
+
+	initapp.PrintInitializationSummary(newApp.Name, newApp.AlternateId, newApp.ID, orgID)
+	return nil
+}
+
+func CreateAndPushEmptyEnvFileMonorepo(cmd *cobra.Command, envService *env.EnvService, m manifest.Manifest, orgID, appID, envID, envName, appDir string) error {
+	envFileName, err := env.GetFileName(envName)
+	if err != nil {
+		return err
+	}
+
+	// Use filepath.Join to create the full path including appDir
+	fullEnvPath := filepath.Join(appDir, envFileName)
+
+	// Ensure the directory exists
+	if err := os.MkdirAll(appDir, 0755); err != nil {
+		return fmt.Errorf("failed to create directory %s: %w", appDir, err)
+	}
+
+	err = CreateGitignoredFileMonorepo(cmd, fullEnvPath, envFileName)
+	if err != nil {
+		return err
+	}
+
+	// Build an Env struct from that new empty file
+	envStruct, err := env.GetLocalEncryptedEnv(envName, &appDir, m)
+	if err != nil {
+		return err
+	}
+
+	version := 1
+	envStruct.Version = &version
+
+	if err := envService.PutEnvironmentEnv(orgID, appID, envID, m.SecretKeyId, envStruct); err != nil {
+		if !errors.Is(err, errors.ErrConflict) {
+			return err
+		}
+		envStruct, err = envService.GetEnvironmentEnv(orgID, appID, envID, &m.SecretKeyId, nil)
+		if err != nil {
+			return err
+		}
+		version = *envStruct.Version
+	}
+
+	db, err := database.Restore()
+	if err != nil {
+		return err
+	}
+
+	newEnvDcrypted, err := envStruct.DecryptData(m.GetSecretKey())
+	if err != nil {
+		return err
+	}
+
+	secretKey := database.SecretKey{
+		ProjectId: *m.ProjectId,
+		AppId:     *m.AppId,
+		EnvName:   envName,
+	}
+
+	if err := db.UpsertSecret(secretKey, newEnvDcrypted, version); err != nil {
+		return fmt.Errorf("failed to save local environment: %w", err)
+	}
+
+	return nil
+}
+
+func CreateGitignoredFileMonorepo(cmd *cobra.Command, fullPath, fileName string) error {
```

Comment:
This code looks nearyl identical to the CreateGitignoredFile I commented on before. I think it'd make sense to consolidate it in gitutils.

Reply:
I would like to have the monorepo one and the normal one. In the future, they could do more different things.